context/app/static/js/components/workspaces/AddDatasetsTable/AddDatasetsTable.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Stack from '@mui/material/Stack';
+import TextField from '@mui/material/TextField';
+import Autocomplete, { AutocompleteRenderInputParams } from '@mui/material/Autocomplete';
+import InputAdornment from '@mui/material/InputAdornment';
+import Typography from '@mui/material/Typography';
+
+import { UseDatasetsAutocompleteReturnType, SearchAheadHit } from './hooks';
+import WorkspaceDatasetsTable from '../WorkspaceDatasetsTable';
+
+function DatasetOption(props: React.HTMLAttributes<HTMLLIElement>, option: SearchAheadHit) {
+  const {
+    _source: { hubmap_id, assay_display_name, origin_samples_unique_mapped_organs },
+  } = option;
+  return (
+    <li {...props}>
+      <div>
+        <Typography variant="subtitle1">{hubmap_id}</Typography>
+        <Typography variant="body2">
+          {[origin_samples_unique_mapped_organs, assay_display_name].filter((f) => f !== undefined).join(' | ')}
```

Comment:
Should we consider [using `Boolean` as the filter function here?](https://stackoverflow.com/questions/29358815/using-boolean-as-the-argument-to-filter-in-javascript) 

```suggestion
          {[origin_samples_unique_mapped_organs, assay_display_name].filter(Boolean).join(' | ')}
```